src/cv_2.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Конструктор резюме</title>
+    <link rel="stylesheet" href="./styles/reset.css">
+    <link rel="stylesheet" href="./styles/main.css">
+</head>
+<body>
+    <section class="blue">
+        <header class="blocking_heading">
+            <h1 class="h1 white">Конструктор резюме</h1>
+            <nav>
+                <a class="menu_btn input_head" href="cv_1.html">Фото</a>
+                <a class="menu_btn input_head active" href="cv_2.html">О себе</a>
+                <a class="menu_btn input_head" href="cv_3.html">Образование</a>
+                <a class="menu_btn input_head" href="cv_4.html">Опыт</a>
+                <a class="menu_btn input_head" href="cv_5.html">Дополнительно</a>
+            </nav>
+        </header>
+    </section>
+    <section class="blocking main">
+        <div class="ltl_block">
+            <div>
+                <h2 class="h2">Персональные данные</h2>
+                <p>Укажите потенциальным работодателям актуальные контактные данные. Это будет вашим первым шагом к взаимодействию с менеджером по найму, поэтому крайне важно, чтобы информация была точной и актуальной.</p>
+            </div>
+            <div class="input_line">
+                <div class="input_block">
+                    <label for="name" class="input_head">Имя</label>
+                    <input class="input" id="name" placeholder="Константин">
+                </div>
+                <div class="input_block">
+                    <label for="name" class="input_head">Фамилия</label>
```

Comment:
Небольшая опечатка, из-за которой при клике на этот label фокус будет выставлен на другое поле – name.
```suggestion
                    <label for="surname" class="input_head">Фамилия</label>
```